avm/res/databricks/access-connector/main.bicep

```diff
@@ -28,7 +28,9 @@ var formattedUserAssignedIdentities = reduce(map((managedIdentities.?userAssigne
 var identity = !empty(managedIdentities) ? {
   type: (managedIdentities.?systemAssigned ?? false) ? (!empty(managedIdentities.?userAssignedResourceIds ?? {}) ? 'SystemAssigned,UserAssigned' : 'SystemAssigned') : (!empty(managedIdentities.?userAssignedResourceIds ?? {}) ? 'UserAssigned' : null)
   userAssignedIdentities: !empty(formattedUserAssignedIdentities) ? formattedUserAssignedIdentities : null
-} : null
+} : {
```

Comment:
Hey @eriqua,
unfortunately, I can't tell you. Maybe it's just something that wasn't tested before. In any case I see an increasing amount of linter warnings regarding the `null` implementation which really is a pitty since `null` is great.
For this module we don't have many options. It fundamtenally does not work without the value. That said, I should also mention that we have like 5 or 6 different variations of the identity provider so it's only seemingly consistent. The approach aligns, but if needed, modules have individual adjustments (for example, whether they support user-assigned identities or not, in combination with system-assigned identities, or not, etc. etc.).

Reply:
If that is the case I guess it's fair to say that the implementation consistency is to be adapted to the specific resource. In the end, it's the alignment of the interface input that matters the most. I'll then approve the PR